### PR TITLE
Handle refusal event

### PR DESCRIPTION
**Pull request by AdeHarrison**
## Motivation and Context
To test a Refusal message

## What has changed
Added new acceptance test

## How to test?
[as per](https://github.com/ONSdigital/census-rm-case-processor/pull/30)

## Links
[Trello](https://trello.com/c/29ZsK7ci/825-rmc-197-handle-refusal-received-event-rev13-8)

### CODE REVIEW

**Comment by AdeHarrison** [comment]
[RMC-197 Handle Refusal Received Event [REV13] (8)](https://trello.com/c/29ZsK7ci/825-rmc-197-handle-refusal-received-event-rev13-8)

**Review by ryangrundy7 — DISMISSED** [review]
Got some flake 8 errors when trying to run make test

```
./acceptance_tests/features/steps/refusal.py:22:9: E122 continuation line missing indentation or outdented
./acceptance_tests/features/steps/refusal.py:29:9: E122 continuation line missing indentation or outdented
./acceptance_tests/features/steps/refusal.py:55:5: E122 continuation line missing indentation or outdented
make: *** [flake] Error 1
```

File: acceptance_tests/features/refusal.feature [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Feature: Handle refusal message
+
+  @wip
```
Comment (ryangrundy7) [inline acceptance_tests/features/refusal.feature]: Don't need this wip tag

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
There’s an existing test for the FWMT adapter in fieldwork.feature which sends a baked refusal message and checks the right xml message is to FWMT. Now case-processor is also subscribing to refusals and trying to ingest that baked refusal message but it can’t because the case isn’t set up so it spins on retrying it. I think the `then` from that scenario needs to be included in this new new one and the old scenario deleted.

**Review by jcox-dev — DISMISSED** [review]
I noticed there was still a [utility function](https://github.com/ONSdigital/census-rm-acceptance-tests/blob/705158656efe9ffd6b81fff8a3e83b356b9aeeec/acceptance_tests/features/steps/receipt.py#L72) that was still using the outdated `response_datetime` JSON item (removed as part of https://github.com/ONSdigital/census-rm-pubsub/pull/29). This function can probably be removed as part of this PR.

**Review by ryangrundy7 — APPROVED** [review]
Works with all the PRs 👍

**Review by jcox-dev — APPROVED** [review]
Tests all pass in GKE against the branched services  🍰🎉

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**